pages/splash.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from "../styles/Splash.module.css";
+import Logo from "../components/logo/logo";
+
+export default function SplashPage() {
+  return (
+    <div>
+      <div className={styles.logo}>
+        <Logo logo="bakend" size="big" />
+      </div>
+      <div>
+        <a className={styles.link} href="http://localhost:3000">
```

Comment:
```suggestion
        <a className={styles.link} href="/">
```

To link to Home globally you can just "/".

Reply:
Thank you!!